src/yetibot/core/monitoring.clj

```diff
@@ -0,0 +1,35 @@
+(ns yetibot.core.monitoring
+  (:require [clojure.spec.alpha :as s]
+            [metrics.reporters.riemann :as riemann]
+            [taoensso.timbre :as log]
+            [yetibot.core.config :as config]
+            [yetibot.core.spec :as yspec])
+  (:import [java.util.concurrent TimeUnit]
+           [com.codahale.metrics MetricFilter]))
+
+(s/def ::host ::yspec/non-blank-string)
+
+(s/def ::port pos-int?)
+
+(s/def ::riemann (s/keys :req-un [::host ::port]))
+
+(s/def ::config (s/keys :req-un [::riemann]))
+
+(defonce riemann-reporter (atom nil))
+
+(defn start
+  []
+  (log/info "starting monitoring")
```

Comment:
Would be useful if we only log this when config is valid, and if not maybe a debug statement saying monitoring is not configured.

Reply:
Done. Also made the port optional and interval configurable. What do you think?

Reply:
Perfect!